src/editor/index.js

```diff
@@ -145,6 +145,23 @@ registerPlugin( 'newspack-popups-editor', {
 	icon: null,
 } );
 
+// Hide Newspack's Homepage Posts block deduplication toggle when the popup is an overlay.
+registerPlugin( 'newspack-popups-disable-newspack-blocks-deduplication', {
+	render: function HideDeduplicationToggle() {
+		const { isOverlay } = useSelect( select => {
+			const { getEditedPostAttribute } = select( 'core/editor' );
+			return {
+				isOverlay: getEditedPostAttribute( 'meta' )?.placement === 'center',
```

Comment:
We should check for any of the valid overlay placements as defined here: https://github.com/Automattic/newspack-popups/blob/master/includes/class-newspack-popups-model.php#L19

Reply:
Whoops! Thank you for catching that, fixed in 849bad6215f3002c5cfbc4613d66229f6f06216d